definitions/npm/moment_v2.x.x/flow_v0.25.x-/moment_v2.x.x.js

```diff
@@ -118,30 +118,51 @@ declare class moment$MomentDuration {
 }
 declare class moment$Moment {
   static ISO_8601: string;
+  static (): moment$Moment;
   static (
-    initDate: ?Object | number | Date | Array<number> | moment$Moment | string
+    initDate: moment$MomentOptions | number | Date | Array<number> | moment$Moment | string
   ): moment$Moment;
+  static (string: string): moment$Moment;
+  static (string: string, format: string | Array<string>): moment$Moment;
   static (
-    string?: string,
-    format?: string | Array<string>,
-    strict?: boolean
+    string: string,
+    format: string | Array<string>,
+    strict: boolean
+  ): moment$Moment;
+  static (
+    string: string,
+    format: string | Array<string>,
+    locale: string
   ): moment$Moment;
   static (
-    string?: string,
-    format?: string | Array<string>,
-    locale?: string,
-    strict?: boolean
+    string: string,
+    format: string | Array<string>,
+    locale: string,
+    strict: boolean
   ): moment$Moment;
   static unix(seconds: number): moment$Moment;
   static utc(): moment$Moment;
-  static utc(number: number | Array<number>): moment$Moment;
   static utc(
-    str: string,
-    str2?: string | Array<string>,
-    str3?: string
+    initDate: moment$MomentOptions | number | Date | Array<number> | moment$Moment | string
+  ): moment$Moment;
+  static utc(string: string): moment$Moment;
```

Comment:
Same goes for this line. Let me know if I'm missing something!

Reply:
Oh yes, you're right about that. I'll change these both.